patterns/vsi-extension/variables.tf

```diff
@@ -84,8 +84,8 @@ variable "existing_kms_instance_guid" {
 
 variable "skip_iam_authorization_policy" {
   type        = bool
-  description = "Set to `true` to skip the creation of an IAM authorization policy that permits all storage blocks to read the encryption key from the KMS instance. If set to `false` (and creating a policy), specify the GUID of the KMS instance in the `existing_kms_instance_guid` variable."
-  default     = false
+  description = "Set to `true` to skip the creation of an IAM authorization policy that permits all storage blocks to read the encryption key from the KMS instance. If set to `false` (and creating a policy), specify the GUID of the KMS instance in the `existing_kms_instance_guid` variable. When you are using the same KMS instance from the landing zone VPC, it may be possible to set a flag to true in order to skip creating an authorization policy since it has already been established by the VPC DA."
+  default     = true
```

Comment:
The wording here doesn't make sense. The description says `it may be possible to set a flag a flag to true` however the default value is true. I think the point we are trying to make is that by default the Landing Zone VPC DA will create this policy, and that is why we default it to true. Might need @SirSpidey to review how best to word this.
I'm wondering if we should be exposing it all, but I guess it is no harm to have it as an option with the default value to true so its skipped? Any thoughts on this @toddgiguere ?

I also just noticed that `existing_kms_instance_guid` has no default value, so that will make it a required variable, however it seems its only required if `skip_iam_authorization_policy` is false

Reply:
here's one suggestion: 

"By default (`true`), the Landing Zone VPC creates an IAM authorization policy that permits all storage blocks to read the encryption key from the KMS instance. Set to `false` to create the authorization policy in a different KMS instance, and specify the GUID of the KMS instance in the `existing_kms_instance_guid` variable."

Reply:
@ocofaigh ,
even with the `skip_iam_authorization_policy ` set to false, we require the `existing_kms_instance_guid ` to encrypt the block volumes. 
Another option is to remove the default value for `skip_iam_authorization_policy ` as well, so that the user can decide whether to create the auth policy or not.

Reply:
@Aashiq-J OK so `existing_kms_instance_guid` is needed. Can it be looked up from schematics if user picks that approach though?

I think maybe we leave the variable in, but default it to `true` since SLZ VPC DA creates it, but lets update the description to make that clear

Reply:
@ocofaigh ,
Sorry about the confusion, `existing_kms_instance_guid ` is not required when we don't have to create an auth policy. `boot_volume_encryption_key` is the only variable required to encrypt boot volume if we skip the auth policy creation.

So, we can set null as default value for `existing_kms_instance_guid` and have a validation to check for value when `skip_iam_authorization_policy ` is set to `false`.